active/tab/tabcontrol.md

```diff
@@ -0,0 +1,580 @@
+    static Windows.UI.Xaml.DependencyProperty TabStripFooterContentProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty TabStripFooterContentTemplateProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty TabStripHeaderProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty TabStripHeaderTemplateProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty IsAddTabButtonVisibleProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty AddTabButtonCommandProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty AddTabButtonCommandParameterProperty{ get; };
+
+    // TODO: Update to "TabItem*"
+    static Windows.UI.Xaml.DependencyProperty TabItemsSourceProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty TabItemsProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty TabItemTemplateProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty TabItemTemplateSelectorProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty SelectedIndexProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty SelectedTabItemProperty{ get; };
+}
+
+[WUXC_VERSION_PREVIEW]
+[webhosthidden]
+unsealed runtimeclass TabViewItem : Windows.UI.Xaml.Controls.ListViewItem
+{
+    TabViewItem();
+
+    Object Header{ get; set; };
+
+    Windows.UI.Xaml.DataTemplate HeaderTemplate{ get; set; };
+
+    Windows.UI.Xaml.Controls.IconElement Icon{ get; set; };
+
+    [MUX_DEFAULT_VALUE("true")]
+    [MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
+    Boolean IsCloseable{ get; set; };
+
+    static Windows.UI.Xaml.DependencyProperty HeaderProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty HeaderTemplateProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty IconProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty IsCloseableProperty{ get; };
+
+    // TODO: Add
+    event Windows.Foundation.TypedEventHandler<TabViewItem, TabViewTabClosingEventArgs> TabClosing;
+}
+
+}
+
+namespace MU_XCP_NAMESPACE
+{
+
+[WUXC_VERSION_PREVIEW]
+[webhosthidden]
+unsealed runtimeclass TabViewListView : Windows.UI.Xaml.Controls.ListView
+{
+    TabViewListView();
+}
+
+}
+
+namespace MU_XAP_NAMESPACE
+{
+
+[WUXC_VERSION_PREVIEW]
+[webhosthidden]
+unsealed runtimeclass TabViewAutomationPeer : Windows.UI.Xaml.Automation.Peers.FrameworkElementAutomationPeer
+{
+    TabViewAutomationPeer(MU_XC_NAMESPACE.TabView owner);
+}
+
+[WUXC_VERSION_PREVIEW]
+[webhosthidden]
+unsealed runtimeclass TabViewItemAutomationPeer : Windows.UI.Xaml.Automation.Peers.ListViewItemAutomationPeer
+{
+    TabViewItemAutomationPeer(MU_XC_NAMESPACE.TabViewItem owner);
+}
+
+}
+```
+
+# Appendix
+<!-- Anything else that you want to write down for posterity, but 
+that isn't necessary to understand the purpose and usage of the API.
+For example, implementation details. -->
+
+![Position of TabStripHeader and TabStripFooter](images/tab-control-parts-1.PNG)
+
+![Parts of a tab item](images/tab-control-parts-2.PNG)
```

Comment:
These diagrams are so important I think they need to be moved out of the Appendix to main body. In particular, the breakdown of TabViewItem.Header / TabViewItem.Content is essential for understanding the `CreateNewTabFromDocument` sample function.

Reply:
Consider: Move this up